src/routes/accountOpeningRequest.ts

```diff
@@ -158,21 +146,23 @@ export const retrieveAccountOpeningRequest = async (
     let customerType = CustomerType.PERSON;
     let entityId: string;
     let entity: MockPerson | MockBusiness;
+    let handlers = getHandlers(customerType);
```

Comment:
What if we were to move the getHandler into the if/else statement so it's only called once 😄 

```suggestion
    let handlers;
```